tests/app_test.py

```diff
@@ -0,0 +1,30 @@
+import requests
+import unittest
+import urllib
+from flask import Flask
+from flask_testing import LiveServerTestCase
```

Comment:
I got an error mentioned [here](https://stackoverflow.com/questions/64305197/getting-a-cant-pickle-local-object-liveservertestcase-error-when-trying-to)
I'm not sure if this applies to other machines but `multiprocessing.set_start_method("fork")` did help me out

Reply:
I am currently running the test in an environment with Anaconda. `$ pytest` told me that all the tests can be passed. what environment are you using now?

My console output:
```
$ pytest
=================================================== test session starts ====================================================
platform darwin -- Python 3.6.6, pytest-6.2.4, py-1.10.0, pluggy-0.13.1
rootdir: /Users/myname/Desktop/API_deployment
collected 3 items                                                                                                          

tests/app_test.py ...                                                                                                [100%]

==================================================== 3 passed in 0.53s =====================================================

```

Reply:
I'm using Mac terminal with python 3.9

Reply:
So I tried running in conda and it works fine, so it might be something wrong with my base config...